code/game/objects/machinery/mortar.dm

```diff
@@ -1,6 +1,9 @@
 //The Marine mortar, the T-50S Mortar
 //Works like a contemporary crew weapon mortar
 
+#define MORTAR 1
```

Comment:
In general, you want your vars /defines to be as verbose as possible.

Here, TALLY_MORTAR would be a better name